marsha/core/base_models.py

```diff
@@ -120,7 +143,8 @@ def _get_expected_field_annotation(cls, field: models.Field) -> Tuple[type, str]
         """
         if isinstance(field, models.ManyToManyField):
             return M2MType[field.related_model], '%s["%s"]' % (  # type: ignore
```

Comment:
Be careful they are still some % remaining throughout the code base, to be replaced by format(...)

Reply:
Ok I'll do another commit for code not related to this PR